src/library.rs

```diff
@@ -1006,7 +1013,7 @@ impl Library {
                 let full_name = format!("{}.{}", namespace_name, name);
```

Comment:
The list of names above should probably also contain `Interface`. But for what reason is `Type` included there?

Reply:
I don't know actually. On `atk`, all "*Type" are enumerations at least. Maybe it's not the case on other crates?

Reply:
Agree about `Interface`

Reply:
I added it.

Reply:
So should we remove `Type` here?

Reply:
Yes? If not, why not? :)